src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  useCallback,
+  useContext,
+  useMemo,
+} from 'react';
+import { TodosContext } from '../../TodoContext/TodosContext';
+import { Status } from '../../Enum/Status';
+import { Todo } from '../../Interface/Todo';
+
+export const TodoFooter = () => {
+  const {
+    todos,
+    setTodos,
+    filterBy,
+    setFilterBy,
+  } = useContext(TodosContext);
+
+  const uncomplitedTodos = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+
+  const deleteTodos = useCallback(() => {
+    setTodos((currentTodos: Todo[]) => (
+      currentTodos.filter((todo: Todo) => !todo.completed)
+    ));
+  }, [todos]);
+
+  return (
+    <>
+      {todos.length > 0 && (
+        <footer className="footer">
+          <span className="todo-count" data-cy="todosCounter">
+            {`${uncomplitedTodos} items left`}
+          </span>
+
+          <ul className="filters" data-cy="todosFilter">
+            <li>
+              <a
+                href="#/"
+                className={
+                  Status.ALL === filterBy
+                    ? 'selected'
+                    : ''
+                }
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally. Fix all cases